lib/application.js

```diff
@@ -308,6 +308,11 @@ Origin.prototype.startServer = function () {
     serverOptions.minimal = true;
   }
   
+  // Check the NodeJS version 
+  if (process.version !== 'v0.10.33') {
+    logger.log('warn', 'Incorrect version of NodeJS required -- this code requires v0.10.33 to run correctly.');
```

Comment:
I think the message could worded a little better, here. I suggest:
"You are running an unsupported version of NodeJS (" + process.version + "). This application has only been fully tested on NodeJS v0.10.33. It is recommended that you use that version instead."